packages/goblint-cil/goblint-cil.1.8.0/opam

```diff
@@ -0,0 +1,67 @@
+opam-version: "2.0"
+synopsis:
+  "A front-end for the C programming language that facilitates program analysis and transformation"
+description: """\
+This is a fork of the 'cil' package needed to build 'goblint'.
+Changes:
+- Proper support for C99, (#9) and VLAs in particular (#5, #7)
+- It uses Zarith instead of the deprecated Num
+- Support for more recent OCaml versions (≥ 4.06)
+- Large integer constants that do not fit in an OCaml int are represented as a string instead of getting truncated
+- Syntactic search extension (#21)
+- Some warnings were made optional
+- Unmaintained extensions (#30) were removed
+- Many bug fixes"""
+maintainer: [
+  "Michael Schwarz <michael.schwarz93@gmail.com>"
+  "Ralf Vogler <ralf.vogler@gmail.com>"
+]
+authors: "gabriel@kerneis.info"
+license: "BSD-3-Clause"
+homepage: "https://cil-project.github.io/cil/"
+bug-reports: "https://github.com/goblint/cil/issues"
+depends: [
+  "ocaml" {>= "4.04.2"}
+  "ocamlfind"
```

Comment:
```suggestion
```
This dependency doesn't seem to be used anymore

Reply:
The configure script from the old days that we still use relies on `ocamlfind` being available, and at least one of the tests actively needs it. In the long run we probably want to drop that dependency, but right now it does not seem to be worth the extra effort.